get.sh

```diff
@@ -115,7 +115,13 @@ getBinaryOpenjdk()
 		fi
 	else 
 		download_url=$CUSTOMIZED_SDK_URL
-		wget -q --no-check-certificate --header 'Cookie: allow-download=1' ${download_url}
+		if [[ "$PLATFORM" == "s390x_zos" ]];
+		then
+			curl -OLJ ${download_url} 
```

Comment:
OLJ options could be removed as it is only required when redirection is needed. Besides some lower curl version doesn't support it.

Reply:
Done.